bench/00_transform/einsum.cu

```diff
@@ -1,7 +1,7 @@
 #include <nvbench/nvbench.cuh>
 #include "matx.h"
 
-#if ENABLE_CUTENSOR
+#if MATX_EN_CUTENSOR
```

Comment:
This should be `#ifdef`. Bug was there prior

Reply:
good catch I didn't notice. updated